docs/content/dns.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "DNS"
+---
+
+If the API server can be accessed by alternative DNS addresses then setting additional SANs on the API server
+certificate is necessary in order for clients to successfully validate the API server certificate.
+
+To enable the API server certificate SANs enable the `dnsvars` and `dnspatch`
+external patches on `ClusterClass`.
```

Comment:
Sorry I'm not understanding the relationship between additional SANs and this DNS image field https://github.com/kubernetes-sigs/cluster-api/blob/c29c5f4d467422784d4f29b704972b4527fa6f94/bootstrap/kubeadm/types/upstreamv1beta3/types.go#L176-L184, can you please clarify it.

Reply:
Copy-paste mistake :sweat: